File: app/src/main/java/com/robotnec/budget/core/mvp/view/CalculatorView.java
package com.robotnec.budget.core.mvp.view;

/**
 * @author zak <zak@swingpulse.com>
 */
public interface CalculatorView extends View {
    void display(String value);

    void displayError();

    void displayDone();

    void close();

    void clearError();

    void clearDone();

    void done(double value);
}

File: app/src/main/java/com/robotnec/budget/core/mvp/view/CalculatorView.kt
package com.robotnec.budget.core.mvp.view

import org.joda.money.Money

/**
 * @author zak zak@swingpulse.com>
 */
interface CalculatorView : View {
    fun display(value: String)

    fun close()

    fun done(money: Money)

    fun showMaxCountReached()
}
